communities.json

```
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "schoology-plus.png",
+      "title": "Schoology Plus",
+      "quote": "We use Discord as a place to discuss potential new features with our community, discuss in progress ideas, and collect user bug reports.",
+      "inviteCode": "PdB8dEn",
```

Do you want to update this to your new vanity url code?
```suggestion
      "inviteCode": "schoologyplus",
```